### PR TITLE
rewrite dotnet examples

### PR DESCRIPTION
**Pull request by mihaildolghintev**

### CODE REVIEW

File: dotnet/requestsApp/SaltEdge.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace SESample
+{
+    class SaltEdge
+    {
+        const int REQUEST_EXPIRES_MINUTES = 3;
+        const string APP_ID = "123";
+        const string SECRET = "123";
```
Comment (alexeisersun) [inline dotnet/requestsApp/SaltEdge.cs:16]: Maybe extract this into a config file? And raise error/ add warning if developers don't configure it properly?

File: dotnet/requestsApp/Program.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            // **********************************
+            // ****** POST request example ******
+            // **********************************
+
+            // This is an example of a post request with a payload. Keep in mind the the serialized payload
+            // is also taken into account when computing the signature.
+            //
+            // var payload = new {
+            //     data = new {
+            //         identifier = "bbbbjjjj12rv1212f1efxchsdhbgv"
```
Comment (alexeisersun) [inline dotnet/requestsApp/Program.cs:30]:
```suggestion
            //         identifier = "your_customer_email@domain.com"
```

File: dotnet/requestsApp/SaltEdge.cs (hunk 1 above)
Comment (mihaildolghintev) [reply dotnet/requestsApp/SaltEdge.cs]: implemented

File: dotnet/requestsApp/Program.cs (hunk 2 above)
Comment (mihaildolghintev) [reply dotnet/requestsApp/Program.cs]: sure)

**Pull request merged**